image/image.go

```diff
@@ -29,30 +29,30 @@ func IDFromDigest(digest digest.Digest) ID {
 
 // V1Image stores the V1 image configuration.
 type V1Image struct {
-	// ID a unique 64 character identifier of the image
-	ID string `json:"id,omitempty"`
+	// ID is a unique 64 character identifier of the image
+	ID              string `json:"id,omitempty"`
```

Comment:
Is this correct? Did you manually make these whitespace changes? running `gofmt` on this file gives me;

```patch
diff --git a/image/image.go b/image/image.go
index 75dc92e..58e5a7e 100644
--- a/image/image.go
+++ b/image/image.go
@@ -30,29 +30,29 @@ func IDFromDigest(digest digest.Digest) ID {
 // V1Image stores the V1 image configuration.
 type V1Image struct {
        // ID is a unique 64 character identifier of the image
-       ID              string `json:"id,omitempty"`
+       ID string `json:"id,omitempty"`
        // Parent id of the image
-       Parent          string `json:"parent,omitempty"`
+       Parent string `json:"parent,omitempty"`
        // Comment user added comment
-       Comment         string `json:"comment,omitempty"`
+       Comment string `json:"comment,omitempty"`
        // Created timestamp when image was created
-       Created         time.Time `json:"created"`
+       Created time.Time `json:"created"`
        // Container is the id of the container used to commit
-       Container       string `json:"container,omitempty"`
+       Container string `json:"container,omitempty"`
        // ContainerConfig is the configuration of the container that is committed into the image
        ContainerConfig container.Config `json:"container_config,omitempty"`
        // DockerVersion specifies version on which image is built
-       DockerVersion   string `json:"docker_version,omitempty"`
+       DockerVersion string `json:"docker_version,omitempty"`
        // Author of the image
-       Author          string `json:"author,omitempty"`
+       Author string `json:"author,omitempty"`
        // Config is the configuration of the container received from the client
-       Config          *container.Config `json:"config,omitempty"`
+       Config *container.Config `json:"config,omitempty"`
        // Architecture is the hardware that the image is build and runs on
-       Architecture    string `json:"architecture,omitempty"`
+       Architecture string `json:"architecture,omitempty"`
        // OS is the operating system used to build and run the image
-       OS              string `json:"os,omitempty"`
+       OS string `json:"os,omitempty"`
        // Size is the total size of the image including all layers it is composed of
-       Size            int64 `json:",omitempty"`
+       Size int64 `json:",omitempty"`
 }

 // Image stores the image configuration
@@ -65,7 +65,7 @@ type Image struct {
        OSFeatures []string  `json:"os.features,omitempty"`

        // rawJSON caches the immutable JSON associated with this image.
-       rawJSON    []byte
+       rawJSON []byte

        // computedID is the ID computed from the hash of the image config.
        // Not to be confused with the legacy V1 ID in V1Image.
@@ -113,17 +113,17 @@ func (img *Image) MarshalJSON() ([]byte, error) {
 // History stores build commands that were used to create an image
 type History struct {
        // Created timestamp for build point
-       Created       time.Time `json:"created"`
+       Created time.Time `json:"created"`
        // Author of the build point
-       Author        string `json:"author,omitempty"`
+       Author string `json:"author,omitempty"`
        // CreatedBy keeps the Dockerfile command used while building image.
-       CreatedBy     string `json:"created_by,omitempty"`
+       CreatedBy string `json:"created_by,omitempty"`
        // Comment is custom message set by the user when creating the image.
-       Comment       string `json:"comment,omitempty"`
+       Comment string `json:"comment,omitempty"`
        // EmptyLayer is set to true if this history item did not generate a
        // layer. Otherwise, the history item is associated with the next
        // layer in the RootFS section.
-       EmptyLayer    bool `json:"empty_layer,omitempty"`
+       EmptyLayer bool `json:"empty_layer,omitempty"`
 }

 // Exporter provides interface for exporting and importing images
```